tests/vmi_configuration_test.go

```diff
@@ -353,6 +353,54 @@ var _ = Describe("Configurations", func() {
 
 		})
 
+		Context("with usb controller", func() {
+			It("should start the VMI with usb controller", func() {
+				vmi := tests.NewRandomVMIWithEphemeralDisk(tests.ContainerDiskFor(tests.ContainerDiskAlpine))
+				_true := true
+				vmi.Spec.Domain.Features = &v1.Features{
+					USBDevice: &v1.FeatureState{
+						Enabled: &_true,
+					},
+				}
+				By("Starting a VirtualMachineInstance")
+				vmi, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(vmi)
+				Expect(err).ToNot(HaveOccurred(), "should start vmi")
+				tests.WaitForSuccessfulVMIStart(vmi)
+
+				By("Expecting the VirtualMachineInstance console")
+				expecter, err := tests.LoggedInAlpineExpecter(vmi)
+				Expect(err).ToNot(HaveOccurred(), "should get console")
+				defer expecter.Close()
+
+				By("Checking the number of usb under guest OS")
+				_, err = expecter.ExpectBatch([]expect.Batcher{
+					&expect.BSnd{S: "ls -l /sys/bus/usb/devices/usb* | wc -l\n"},
```

Comment:
What we're really trying to check here is that the USB controller is enabled, so it might be safer to check for a value greater than zero instead of checking for exactly 2 devices.